minio/xml.py

```diff
@@ -80,20 +92,43 @@ def findtext(element, name, strict=False):
     return element.text or ""
 
 
-def unmarshal(cls, xmlstring):
+K = TypeVar("K")
```

Comment:
Have meaningful name here

Reply:
this is generic type parameter, it's custom to be named as `T` `K` `J`。